### ISSUE TITLE
RDPWrap Saying Not Supported

### ISSUE THREAD

**Issue by miamilamiw** [DuoStream/Duo#82]
After latest Windows 11 2H23 update build 21415865 RDPWrap shows as not supported showing termsrv ver. 10.0.22621.3374

 I have updated RDP.ini with no luck. 

I think because this is not working I am unable to browse web interface in Duo Manager for my started instance and nothing shows in my client Moonlight

I have Patreon version and update with latest Github version

I checked Windows Application logs and Duo has no errors

I have Core Isolation and Memory Integrity off and My Windows Antivirus has exceptions for RDPWrap and Duo directories

**Comment by RemixLivee** [comment]
Download and run [RDPWrapOffsetFinder](https://github.com/llccd/RDPWrapOffsetFinder/releases), then manually copy the information given in the RDP.ini file. After doing this, the text "fully supported" should appear in RDP Wrapper.

**Comment by miamilamiw** [comment]
I ran both 64bit exe's but I don't see an rdp.ini file anywhere that's generated for me to copy the contents?

**Comment by loyejaotdiqr47123** [comment]
[RDPWrap-v1.8.2.zip](https://github.com/DuoStream/Duo/files/14803224/RDPWrap-v1.8.2.zip)
Run update.bat

**Comment by RemixLivee** [comment]
> I ran both 64bit exe's but I don't see an rdp.ini file anywhere that's generated for me to copy the contents?

You have to execute RDPWrapOffsetFinder in a cmd.exe and copy / paste the result. To save you time, here it is:

```
[10.0.22621.3374]
SingleUserPatch.x64   =1
SingleUserOffset.x64  =C3F3
SingleUserCode.x64    =Zero
DefPolicyPatch.x64    =1
DefPolicyOffset.x64   =1C2E5
DefPolicyCode.x64     =CDefPolicy_Query_eax_rcx
LocalOnlyPatch.x64    =1
LocalOnlyOffset.x64   =9D661
LocalOnlyCode.x64     =jmpshort
SLInitHook.x64        =1
SLInitOffset.x64      =29ED0
SLInitFunc.x64        =New_CSLQuery_Initialize

[10.0.22621.3374-SLInit]
bServerSku.x64        =12BF50
bRemoteConnAllowed.x64=12BF64
bFUSEnabled.x64       =12BF74
bAppServerAllowed.x64 =12BF5C
bMultimonAllowed.x64  =12BF68
lMaxUserSessions.x64  =12BF54
ulMaxDebugSessions.x64=12BF70
bInitialized.x64      =12BF4C
```

**Comment by RemixLivee** [comment]
> [RDPWrap-v1.8.2.zip](https://github.com/DuoStream/Duo/files/14803224/RDPWrap-v1.8.2.zip) Run update.bat

I tested it with your ZIP but it still says "Not supported". The only solution that works for me is the one I gave above.

**Comment by loyejaotdiqr47123** [comment]
> > [RDPWrap-v1.8.2.zip](https://github.com/DuoStream/Duo/files/14803224/RDPWrap-v1.8.2.zip)运行update.bat
> 
> 我用你的ZIP测试了它，但它仍然说“不支持”。唯一适合我的解决方案是我上面给出的解决方案。

But I added offsets...
[https://github.com/loyejaotdiqr47123/rdpwrap/commit/88c1f8c997480d6473cb14eac351a7bb3bf520f2](https://github.com/loyejaotdiqr47123/rdpwrap/commit/88c1f8c997480d6473cb14eac351a7bb3bf520f2)

**Comment by RemixLivee** [comment]
It's working after uninstalling, thank you!

**Comment by miamilamiw** [comment]
Ok thanks I get RDP multisession working now however my Moonlight client isn’t detecting the sunshine instance I have setup it was earlier today. 

I am using a local admin account called GameStream and Duo Manager starts the instance but I no longer get the open web admin panel after enabling and restarting Duo Services it is greyed out

**Comment by miamilamiw** [comment]
It was rtx hdr disabling it makes it work

**Comment by Black-Seraph** [comment]
All addressed as of version 1.3.8.

RdpWrap management is all automated now, no need to do anything manually.

Also thanks to @loyejaotdiqr47123 for keeping the RdpWrap.ini offsets up-to-date.
